nebula/ui/components/MZSettingsToggle.qml

```diff
@@ -147,12 +139,12 @@ CheckBox {
     focusPolicy: Qt.StrongFocus
 
     Keys.onReleased: event => {
-        if (event.key === Qt.Key_Return)
+        if (event.key === Qt.Key_Return || event.key === Qt.Key_Space)
             uiPlaceholder.state = uiState.stateDefault;
     }
 
     Keys.onPressed: event => {
-        if (event.key === Qt.Key_Return || event.key === Qt.Key_Space)
```

Comment:
do we have guidance from a11y that this is correct or do we get the Key_Space click for free somehow?

Reply:
We get space for free. this change was merely to fix ui states